isort/format.py

```diff
@@ -27,24 +28,27 @@ def format_natural(import_line: str) -> str:
     return import_line
 
 
-def show_unified_diff(*, file_input: str, file_output: str, file_path: str) -> None:
+def show_unified_diff(*, file_input: str, file_output: str, file_path: Optional[Path]) -> None:
+    file_name = '' if file_path is None else str(file_path)
+    file_mtime = str(datetime.now() if file_path is None
+                     else datetime.fromtimestamp(file_path.stat().st_mtime))
+
     unified_diff_lines = unified_diff(
         file_input.splitlines(keepends=True),
         file_output.splitlines(keepends=True),
-        fromfile=file_path + ':before',
-        tofile=file_path + ':after',
-        fromfiledate=str(datetime.fromtimestamp(os.path.getmtime(file_path))
-                         if file_path else datetime.now()),
+        fromfile=file_name + ':before',
+        tofile=file_name + ':after',
+        fromfiledate=file_mtime,
         tofiledate=str(datetime.now())
     )
     for line in unified_diff_lines:
         sys.stdout.write(line)
 
 
-def ask_whether_to_apply_changes_to_file(file_path: str) -> bool:
+def ask_whether_to_apply_changes_to_file(fpath: str) -> bool:
```

Comment:
Curious, why the switch from `file_path` to `fpath`?

Reply:
I think I did it before extracting the function out to discern between variables. Doesn't really matter, can roll it back if you want.

Reply:
I went ahead and changed it to `file_path` just because of my personal preference to spell out words, rather than use abbreviations - but agree completely it doesn't matter much. Especially in the context of all the great and very welcome improvements in this pull request :).